src/utils/translation.ts

```diff
@@ -91,6 +96,7 @@ export async function init() {
         id: ["id"],
         gu: ["gu"],
         vi: ["vi"],
+        ta:["ta"],
```

Comment:
Please use `npm run fmt` to fix formatting problems

Reply:
I fixed This